src/citrine/resources/team.py

```diff
@@ -99,37 +101,81 @@ def list_members(self) -> List[TeamMember]:
                                             version=self._api_version)["users"]
         return [TeamMember(user=User.build(m), team=self, actions=m["actions"]) for m in members]
 
-    def remove_user(self, user_id: Union[str, UUID]) -> bool:
+    def get_member(self, user_id: Union[str, UUID, User]) -> TeamMember:
         """
-        Remove a User from a Team.
+        Get a particular member in the current team.
+
+        May require admin privileges depending on which user is being requested.
```

Comment:
Informational question: do you know what determines if you need admin privileges or not?

Reply:
I believe it's whether you are just asking about yourself, or asking about other users in the group.  Not confident in that answer.